images/java/Dockerfile.standalone

```diff
@@ -0,0 +1,11 @@
+FROM whanos-java:latest
```

Comment:
Same as the C, this should be a builder and the after the build you should do a FROM java without the sdk, maven and all of that. With just the jvm.